### PR TITLE
feat:CO-64 : widget styling as per latest design

### PR DESCRIPTION
**Pull request by arjun-at-roam**
## PR Area

- [x] Frontend

## Type of change

_What kind of change does this PR introduce?_

- [x] Feature

## Description

* Logo, "Learn more" button, and close button has fixed vertical position even when widget title is over multiple lines.
* This PR also fixes other minor variations from the new design
* New design
image
* Actual
image
image
image

* Regression testing
  * Design
image
image
  * Actual
image
image

## Security & risks

- [ ] Adds new third party libraries
- [ ] Adds new `.env` variables
- [ ] Integrates with Third-party apis (dependency must be resilient)
- [ ] Has specific [OWASP security concerns](https://owasp.org/www-project-top-ten/) - list them: (also see [Fixing OWASP Top 10](https://nodegoat.herokuapp.com/tutorial))

## Pre-publish checklist

- [x] I have assigned myself to the ticket
- [x] I have performed a self-review of my code
- [x] I have performed relevant browser testing
- [ ] I have performed keyboard testing of new or modified interactive components

## Reviewer checklist

- [ ] I have run the code (if applicable)
- [ ] I have reviewed and run any tests (if applicable)

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM!

File: public/demo.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h2>AU default widget</h2>
     <script src="./humm-widgets-cards-au.umd.js?productPrice=400&merchantId='aaa143af-a96b-4887-bcd2-85960da544ea'"></script>
 
     <h2>AU default dark widget</h2>
-    <script src="./humm-widgets-cards-au.umd.js?productPrice=400&darkMode='true'"></script>
+    <script src="./humm-widgets-cards-au.umd.js?productPrice=400&merchantId='2b1bd959-5c95-45d7-9365-5844e270a2b6'&darkMode='true'"></script>
```
Comment (arjun-at-roam) [inline public/demo.html:17]: this was overwritten by a merge. It is a mandatory field.

File: src/components/buttons/ButtonClose.vue [hunk 2]
```diff
@@ -34,6 +34,7 @@ export default defineComponent({
 <style scoped lang="scss">
 .button {
   border: 0;
+  margin-top: 5px;
```
Comment (arjun-at-roam) [inline src/components/buttons/ButtonClose.vue:37]: Giving the close button has a fixed height from the top

File: src/styles/cards.scss [hunk 3]
```diff
@@ -22,6 +22,7 @@
 
   &--logo {
     width: 75px;
+    margin-top: 3px;
```
Comment (arjun-at-roam) [inline src/styles/cards.scss:25]: Giving the card logos button has a fixed height from the top
image

File: src/styles/widget.scss [hunk 4]
```diff
@@ -21,52 +21,44 @@
...
+    line-height: 14px;
   }
 
   &__iconbird {
-    margin: 0;
-
-    @media (min-width: 430px) {
-      margin: 9px 0 0 0;
-    }
+    margin: 5px 0 0 0;
```
Comment (arjun-at-roam) [inline src/styles/widget.scss:61]: The logo gets a fixed height from top
image

**Review by antoninroam — APPROVED** [review]

**Review by jpyepez-roam — APPROVED** [review]
Looks great! Self-comments are really helpful, thanks for adding them.

**Pull request merged**